### DISCUSSION TITLE
Issues not shown on user rank card

### DISCUSSION THREAD

**Discussion by shreya-oeo** [anuraghazra/github-readme-stats#2961] [Q&A] [answered]
### Describe the bug

total issues are not shown on https://github-readme-stats.vercel.app/api?username=aulolua&theme=tokyonight&show_icons=true&hide_border=true&count_private=true&include_all_commits=true

### Expected behavior

it should show total issues count > 0

### Screenshots / Live demo link

https://github-readme-stats.vercel.app/api?username=aulolua&theme=tokyonight&show_icons=true&hide_border=true&count_private=true&include_all_commits=true

**Comment by rickstaa** [comment, marked as answer]
@aulolua we use caching to decrease server load (See https://github.com/anuraghazra/github-readme-stats#common-options). You can use a random string to break this cache and check it works.

[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=aulolua&random=2342341)](https://github.com/anuraghazra/github-readme-stats)

```md
[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=aulolua&random=2342341)](https://github.com/anuraghazra/github-readme-stats)
```
